apps/src/applab/ScreenSelector.jsx

```diff
@@ -52,6 +39,10 @@ class ScreenSelector extends React.Component {
   };
 
   render() {
+    if (!this.props.hasDesignMode) {
+      return null;
+    }
```

Comment:
just an FYI that i ended up changing the logic here. previously, we were using CSS to hide the select element if `!this.props.hasDesignMode`, which caused unit tests to fail because they were checking the styles on that element. instead of using CSS to hide the element, we should just not render it at all. i did some manual testing and don't think this will cause any weird regressions.